site/src/components/keyboard-controls/KeyboardControl.tsx

```diff
@@ -9,8 +9,8 @@ export interface KeyboardControlProps {
    * Use "+" to indcate combos - i.e. when more than one key
    * need to pressed at the same time (e.g. "CTRL + A").
    *
-   * Use "/" to separate alternative keys or combos.
-   * E.g. "SPACE / ENTER"
+   * Use "/" or "or" to separate alternative keys or combos.
```

Comment:
I just raised a related PR, didn't see this one (although mine is more focused on styling).

While I was doing it, I noticed this example using a comma to separate: https://github.com/jpmorganchase/salt-ds/blob/main/site/docs/components/toast/accessibility.mdx. Should this be updated to use a slash or 'or', or should we add a comma as a separator? (A lot of ors in one sentence there)

Reply:
Right now (unless this PR is merged), only `/` and `+` are considered separators. They're used to break up the string provided to the `keyOrCombos` prop, so that the actual names of keys can be wrapped in `<kbd>` tags in the output.

For example, `SPACE / ENTER` would end up being rendered as `<kbd>SPACE</kbd> / <kbd>ENTER</kbd>.

So, if you're doing `keyOrCombos="Tab, Tab + Shift"` you'll end up with `<kbd>Tab, Tab</kbd> + <kbd>Shift</kbd>`, which is not what you want.

Fro your PR I would therefore suggest just swapping the comma for a slash for now. If/when this PR is merged, the slash can eventually be replaced by the word "or".

Reply:
Ah sorry, just took a closer look at your PR and I see that you added comma to the list of separators.

Based on a chat @mikearildbrown and I had the other day, we decided that using the work "or" would be clearer though, so maybe you can swap that change for the one I was doing in this PR (i.e. allow "or" to be a separator instead of a comma) and then fixup any MDX files that might be using commas. I can then close this PR since it'll be redundant.

Reply:
Okay cool - makes sense. The lack of separation became more noticeable with the styling. Happy to just get this PR in first and update mine.